### ISSUE TITLE
Header names are converted to lower case

### ISSUE THREAD

**Issue by sgoendoer** [a85/POSTMan-Chrome-Extension#6394]
**Describe the bug**
When sending requests with (custom) HTTP headers, all (custom and non-custom) header names are converted to lower case!!! Applications that rely on case-sensitive values (and/or header names) hence cannot work with this.

**To Reproduce**
1. Start Postman app, create any request, add any case-sensitive header.
2. Send to server
3. Headers are received as lower-case-only on server-side

**Expected behavior**
Upper case header names should remain upper case! Any HTTP header (name and value) sent by Postman app should be received the way it was meant. I.e.:

`MyHeader: val`

should be NOT sent/received as 

`myheader: val`

Which is the case as of now.

RFC 2616 (https://tools.ietf.org/html/rfc2616) clearly states that (at least certain) header field names are to be capitalized. E.g.: Section 14.23, which defines the "Host-header" as:

`Host = "Host" ":" host [ ":" port ] ;`

Even though RFC 7230 states that header names are case-insensitive, this just means that headers of different capitalization should be treated as if they were the same - NOT that they should be converted...

**App information (please complete the following information):**
 - App Type Desktop App v 7.11.0
 - OS: macOS 10.14.4

**Additional context**
Has been reported here: https://github.com/postmanlabs/postman-app-support/issues/4113 in 2018, but still not fixed.

**Comment by shamasis** [comment]
Tracking this already at https://github.com/postmanlabs/postman-app-support/issues/5372. This is something we are ourselves blocked on at a much lower level of our tech stack with an issue being discussed here (https://github.com/nodejs/node/issues/3591#issuecomment-256784252) 😞

**Comment by sgoendoer** [comment]
@shamasis Thanks for your reply but honestly, I don't see how the linked Nodejs-thread is related to this. The linked (and by long closed) thread discusses handling of multiple headers with the same name being present in the same request.

THIS issue means that Postman CHANGES the capitalization of headers. Something that should never ever happen!

I give you that RFC 7230 states that header names are to be treated case insensitive, while still RFC 2616 clearly specifies header names with capitalized letters. I also acknowledge that HTTP/2 requires header names to be all lower case.

Still.

This is not about the equivalence of capitalized oder non-capitalized header names. This is about the fact that Postman CONVERTS header field names to lower case. IMHO, they should not be touched at all unless explicitly specified by the developer!

**Comment by shamasis** [comment]
@sgoendoer Postman does not convert headers to lower-case.

Here is an example wireshark capture that I made a few minutes back... notice the Host header and another header I have underlined that has all sorts of upper and lower case.

<img width="1021" alt="Screenshot 2019-06-17 at 23 20 01" src="https://user-images.githubusercontent.com/232373/59625347-b4520580-9156-11e9-8650-3421e5987335.png">

Perhaps unrelated, but ... the only issue we are tracking is that during sending request, if you have two headers with same key but different capitalisation (like `KeY` and `kEy`), the last one is sent. Postman uses NodeJS to send request and this is a bug of NodeJS and it is their opinion to keep it that way. We are working to patch a workaround for it in the long-term.
